deploy/olm-catalog/ibm-block-csi-operator/1.1.0/ibm-block-csi-operator.v1.1.0.clusterserviceversion.yaml

```diff
@@ -72,26 +74,26 @@ metadata:
             "sidecars": [
               {
                 "name": "csi-node-driver-registrar",
-                "repository": "quay.io/k8scsi/csi-node-driver-registrar",
-                "tag": "v1.2.0",
+                "repository": "registry.redhat.io/openshift4/ose-csi-driver-registrar",
```

Comment:
does it mean that we will store our controller and node images on redhat registry too?

Reply:
it sounds likely, but not sure if so in 1.1.0

Reply:
@27149chen can we resolve this?

Reply:
is it also one click with multi-arch images?

Reply:
you need to cert the images one by one (one image per one project), but I'm not sure if the redhat cert env supports z or power.

Reply:
And we you finish the cert, you just need one click.

Reply:
I'm not sure I follow.
Assuming they support Z, Where do I push the multi-arch manifest? do you have an example from ubiquity?

Reply:
what is the multi-arch manifest? I think we just need to push the images with different arch to the registry which support multi-arch, isn't it?

Reply:
https://docs.docker.com/engine/reference/commandline/manifest/#extended-description
I think there is more to it

Reply:
It seems `registry.connect.redhat.com/ibm` doesn't support this "v2 schema" at the moment, so I guess we'll have to change our operator repository to dockerhub, until they do

Reply:
but as far as i know, we have to push the operator image to redhat registry

Reply:
Then I will ask Scale how they did it